src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts

```diff
@@ -163,16 +165,17 @@ export class CreateScenariosComponent implements OnInit, OnDestroy {
       .pipe(takeUntil(this.destroy$))
       .subscribe((planState) => {
         this.plan$.next(planState.all[planState.currentPlanId!]);
-        this.scenarioConfigId = planState.currentConfigId;
-        this.loadConfig();
+        if (this.scenarioConfigId) {
+          this.loadConfig();
+        }
         this.panelExpanded = planState.panelExpanded ?? false;
       });
 
     // When an area is uploaded, issue an event to draw it on the map.
     // If the "generate areas" option is selected, remove any drawn areas.
     this.formGroups[2].valueChanges.subscribe((_) => {
-      const generateAreas = this.formGroups[2].get('generateAreas');
-      const uploadedArea = this.formGroups[2].get('uploadedArea');
+      const generateAreas = this.formGroups[3].get('generateAreas');
```

Comment:
This is probably outside the scope of this PR, but it would be great if we can name these groups or use something more descriptive than the index to access them

Reply:
Yeah that's definitely on my todo list. Once I have a second after I get the scenario page to load in for already-saved scenarios I want to clean this up